docs/tutorials/1_finetune.md

```diff
@@ -55,15 +138,15 @@ lr_config = dict(
     warmup='linear',
     warmup_iters=500,
     warmup_ratio=0.001,
-    step=[10, 15])
-total_epochs = 20
+    step=[170, 200]) # reduce it
+total_epochs = 210 # reduce it
 ```
 
 ## Use pre-trained model
 
 To use the pre-trained model, the new config add the link of pre-trained models in the `load_from`. The users might need to download the model weights before training to avoid the download time during training.
```

Comment:
I would suggest rewriting the first sentence as:
Users can load a pre-trained model by setting the `load_from` field of the config to the model's path or link.

And the second sentence can be removed.